ProfileManager.js

```diff
@@ -476,10 +500,8 @@ export default class ProfileManager {
     if(typeof profileId !== 'string') {
       throw new TypeError('"profileId" must be a string.');
     }
-    const [profile, agent] = await Promise.all([
-      this.getProfile({id: profileId}),
-      this.getAgent({profileId})
-    ]);
+    const agent = await this.getAgent({profileId});
+    const profile = await this.getProfile({id: profileId});
```

Comment:
Was the parallel fetching here and below not worth it?

Reply:
@davidlehn, yeah, it made it worse because `getProfile` calls `getAgent` internally.